### DISCUSSION TITLE
Native  range slider with react-hook-form

### DISCUSSION THREAD

**Discussion by BekaArabidze** [react-hook-form/react-hook-form#5526] [Q&A] [answered]
Hello,Is there any examples of how to use react-hook-form with native range slider?  I have For examples: {...register("price")} as a prop for my range slider component and changing value onChange, but react-hook-form always gives me 0 . Do you have any ideas? or Do you have any examples?

**Comment by bluebill1049** [comment]
can you share a codesandbox? Happy to get this fixed if there is a problem.

**Comment by bluebill1049** [comment, marked as answer]
it works for me: https://codesandbox.io/s/kind-davinci-j3ifo?file=/src/App.js

**Comment by BekaArabidze** [comment]
Thanks. I got it working.
